lib/features/email/domain/usecases/print_email_interactor.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:core/presentation/state/failure.dart';
+import 'package:core/presentation/state/success.dart';
+import 'package:core/presentation/utils/html_transformer/transform_configuration.dart';
+import 'package:core/utils/app_logger.dart';
+import 'package:dartz/dartz.dart';
+import 'package:tmail_ui_user/features/email/domain/model/email_print.dart';
+import 'package:tmail_ui_user/features/email/domain/repository/email_repository.dart';
+import 'package:tmail_ui_user/features/email/domain/state/print_email_state.dart';
+
+class PrintEmailInteractor {
+  final EmailRepository emailRepository;
+
+  PrintEmailInteractor(this.emailRepository);
+
+  Stream<Either<Failure, Success>> execute(EmailPrint emailPrint) async* {
+    try {
+      yield Right(PrintEmailLoading());
```

Comment:
We should handle it in UI by showing toast.

Reply:
done